test/gateway/index.js

```diff
@@ -10,8 +10,8 @@ const ncp = require('ncp').ncp
 const path = require('path')
 const clean = require('../utils/clean')
 
-describe('HTTP GATEWAY', () => {
-  const repoExample = path.join(__dirname, '../go-ipfs-repo')
+describe('HTTP Gateway', () => {
+  const repoExample = path.join(__dirname, '../js-ipfs-repo')
```

Comment:
Hi @ya7ya. It seems I didn't explain myself well. What I mean is that there should be no "repo copying" either from a previously generated go-ipfs or js-ipfs node. When you spawn a daemon after doing init, it will have its own repo created, that is what should be used for testing.

Reply:
@diasdavid oh, Ma bad, I'll fix that. Sorry.

Reply:
No, my bad for not being clear. Thank you for pushing this :)